gamePlay.js

```diff
@@ -0,0 +1,97 @@
+import { displayRevealedCell, revealWholeGrid, resetDisplay} from "./display.js";
+
+
+export const changeDisplayedCell = function (cell, row, col, gridObject, game) {
+    let currentCell = gridObject.getCell(row, col);
+    if (game.gameOver){
+        return;
+    }
+
+    if (game.isFlaggingOn){
+        let cellObject = gridObject.getCell(row, col);
+        toggleFlagging(cellObject, cell);
+        return;
+    }
+
+    if (currentCell.numMinesAround == 0){
+        gridObject.changeCellProperty(row, col, "isRevealed", true);
+        const revealedCells = gridObject.revealZeros(row, col);
+        revealedCells.forEach(c => displayRevealedCell(c));
+        return;
+    };
+    if (currentCell.isMine){
+        gridObject.changeCellProperty(row, col, "isRevealed", true);
+        revealWholeGrid(gridObject);
+        game.gameOver = true;
+        return;
+    } else {
+        gridObject.changeCellProperty(row, col, "isRevealed", true);
+        displayRevealedCell(currentCell);
+        return;
+    }
+    
+};
```

Comment:
returns are unnecessary here. why?
```suggestion
    if (currentCell.isMine){
        gridObject.changeCellProperty(row, col, "isRevealed", true);
        revealWholeGrid(gridObject);
        game.gameOver = true;
    } else {
        gridObject.changeCellProperty(row, col, "isRevealed", true);
        displayRevealedCell(currentCell);
    }
}
```

Reply:
doesn't apply anymore